src/test/resources/org/broadinstitute/hellbender/tools/copynumber/interval-preprocessor-intervals.interval_list

```diff
@@ -0,0 +1,8 @@
+@HD	VN:1.4	SO:coordinate
```

Comment:
I would name this file to match the tool name.  So `tool-name-goes-here-intervals.interval_list`.

Reply:
Renamed it to preprocess-intervals-test.interval_list